cassandra-diagnostics-connector30/src/integration-test/java/io/smartcat/cassandra/diagnostics/connector/ITConnector.java

```diff
@@ -52,10 +58,14 @@ public void query_is_intercepted_when_connector_is_active() throws InterruptedEx
         session.execute("CREATE KEYSPACE IF NOT EXISTS test_keyspace "
                 + "WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };");
         session.execute("CREATE TABLE IF NOT EXISTS test_keyspace.test_table (uid uuid PRIMARY KEY);");
-        session.execute("SELECT * FROM test_keyspace.test_table");
+        String cql = "SELECT uid FROM test_keyspace.test_table";
+        PreparedStatement prepared = session.prepare(cql);
+        session.execute(cql);
+        session.execute(prepared.bind());
+        Thread.sleep(2000);
         cluster.close();
-        lock.await(2000, TimeUnit.MILLISECONDS);
-        Assert.assertTrue(queryIntercepted);
+        lock.await(60000, TimeUnit.MILLISECONDS);
+        Assert.assertEquals(2, queryInterceptedCnt);
```

Comment:
I had hard time following why 2 and realized that you are testing both prepared and plain string in same method.

Does it make sense to divide into two tests or to rename to `both_prepared_and_string_statement_intercepted_when_connector_is_active` or something like that.